back-end/stampTotalTrend.js

```diff
@@ -0,0 +1,71 @@
+const fs = require("fs"); //File system
+
+let stampss=[]
+
+function iterationObject(obj){
+    let inner_obj = {};    
+
+    for(prop in obj){
+        if(typeof(obj[prop])=="object"){
+            iterationObject(obj[prop]);
+        }
+        else{
+            if(prop=="Stamps"){
+                inner_obj[prop] = obj[prop];
+                stampss.push(inner_obj);
+            }
+        }
+    }
+}
+
+async function stampTotalTrend(req, res) {
+    if(req.query.scheme === undefined){
+        res.sendStatus(400)
+    }
+    else if(req.query.scheme === "002"){
```

Comment:
I don't see why this wouldn't work with other stores. I think an `} else {` here would be just fine